src/Cassandra/Helpers/MultiTargetHelpers.cs

```diff
@@ -0,0 +1,59 @@
+﻿//
+//       Copyright (C) 2019 DataStax Inc.
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+//
+
+using System;
+using System.Reflection;
+
+namespace Cassandra.Helpers
+{
+    internal static class MultiTargetHelpers
+    {
+        public static Assembly GetAssembly(Type type)
+        {
+#if NETSTANDARD1_5
+            return type.GetTypeInfo().Assembly;
+#else
+            return type.Assembly;
```

Comment:
I think `type.GetTypeInfo().Assembly` works on both targets.

Reply:
hmm you're right, not sure why I missed that, changed it!